cmd/kubeseal/main.go

```diff
@@ -22,6 +22,10 @@ import (
 
 	// Register v1.Secret type
 	_ "k8s.io/client-go/pkg/api/install"
+
+	// Register Auth providers
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
Instead of listing them each individually, just import `k8s.io/client-go/plugin/pkg/client/auth`